projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/datetime-picker/datetime-picker-component.spec.ts

```diff
@@ -598,6 +598,16 @@ describe('DatetimePickerComponent', () => {
     const expectedFormattedValue = moment(inputValue).format('YYYY-MM-DDTHH:mm:ss.SSS');
     expect(expectedFormattedValue).toEqual('Invalid date');
   });
+
+  it('should set the correct formatted value in dateControl', () => {
+    fixture.detectChanges();
+    const inputValue = '01-03-2025 11:00 am';
+    component.inputElement.nativeElement.value = inputValue;
+    component.dateTimeEntryFormat = 'DD-MM-YYYY hh:mm a';
+    component.focusOut();
+    const expectedFormattedValue = moment(inputValue, component.dateTimeEntryFormat).format('YYYY-MM-DDTHH:mm:ss.SSS');
+    expect(moment(component.dateControl.value).format('YYYY-MM-DDTHH:mm:ss.SSS')).toEqual(expectedFormattedValue);
+  });
 });
```

Comment:
Think we should have a test where we set the time of the test to DST and non-DST to see if it works in both cases, otherwise it could (not saying it will) break when the clocks change again